### ISSUE TITLE
Add previewPhoto field to post metadata

### ISSUE THREAD

**Issue by justaniles** [justaniles/catilyfe#7]
The previewPhoto will be used for the preview image showed on home page and at the top of reading a post:
```json
previewPhoto: {
    "imageUrl": "string",
    "isHighResolution": "boolean",
    "altText": "string"
}
```
The isHighResolution property will eventually dictate whether to show one of those dank huge images at the top of the post rather than a smaller image, but it doesn't have to be that name

**Comment by petersulucz** [comment]
No repro

**Comment by justaniles** [comment]
Issue still persists for customer

**Comment by petersulucz** [comment]
Customer notoriously complains about non issues

**Comment by justaniles** [comment]
Customer is a VIP, so his complaints receive priority treatment

**Comment by petersulucz** [comment]
Customer has lost his/her mind. We have no VIP program.

**Comment by justaniles** [comment]
You're right, we have no VIP program, but we do have an Ultra Platinum Diamond Plus program. And this customer is in it.

**Comment by petersulucz** [comment]
NO. Thats not the name of the program.
D.E.U.X.
Daringly Extreme Ultimate eXperience

**Comment by justaniles** [comment]
Just enrolled customer in that one too. R/eopening issue.

**Comment by petersulucz** [comment]
# fine.
